admin/server-timing.php

```diff
@@ -105,12 +112,20 @@ static function() {
 		'benchmarking_filters',
 		__( 'Filters', 'performance-lab' ),
 		static function() {
-			perflab_render_server_timing_page_field( 'benchmarking_filters' );
+			perflab_render_server_timing_page_hooks_field( 'benchmarking_filters' );
 		},
 		PERFLAB_SERVER_TIMING_SCREEN,
 		'benchmarking',
 		array( 'label_for' => 'server_timing_benchmarking_filters' )
 	);
+	add_settings_field(
+		'output_buffering',
+		__( 'Output Buffering', 'performance-lab' ),
+		'perflab_render_server_timing_page_output_buffer_checkbox',
+		PERFLAB_SERVER_TIMING_SCREEN,
+		'benchmarking',
```

Comment:
Instead of this, I would advise to put this field into a new, standalone "Output Buffering" settings section on the screen that is displayed on top (since that puts more attention to the field). You could handle that in the same way and appearance as what the "Uploading Files" section in _Settings > Media_ looks like: The settings section heading and below the checkbox, aligned to the left of the screen.

This feedback is primarily based on looking at the screenshot in the PR description. I think bringing this field to the top, in a dedicated section, would ensure it's not missed, and a dedicated section also makes sense since it is not directly tied to the benchmarking hooks fields.

Reply:
I like the idea to move it up, but unfortunately it doesn't look like the Uploading Files section can be used as a model. It's actually a hard-coded settings screen and isn't using the settings API: https://github.com/WordPress/wordpress-develop/blob/7cd891845e5a3f461c3dbc1487467ab5e91c1827/src/wp-admin/options-media.php#L125-L169

Setting fields in sections always include that left table heading cell: https://github.com/WordPress/wordpress-develop/blob/7cd891845e5a3f461c3dbc1487467ab5e91c1827/src/wp-admin/includes/template.php#L1801-L1828

Reply:
Looks like we could do so by not adding the field via `add_settings_field()`

Reply:
Well, I did this in 671c001246bf75ac125523213cf417a4ac732ad5, although duplicating the styles is not great. It looks good though:

![image](https://github.com/WordPress/performance/assets/134745/e7ab702d-68c7-48a7-85ba-2818b27d062c)

Reply:
Thanks, I think that looks solid. While it's not great we have to duplicate those styles, I think it's an acceptable trade-off, particularly as you've already added a comment outlining where those come from, so 👍